openshift/templates/angular-on-nginx/angular-on-nginx-deploy.json

```diff
@@ -18,7 +18,8 @@
       "metadata": {
         "creationTimestamp": null,
         "labels": {
-          "app": "${GROUP_NAME}"
+          "app": "${GROUP_NAME}",
+          "role": "frontend-public"
```

Comment:
Update role to be a var with these values as default

Reply:
made  this change with last push